chart/templates/certificate.yaml

```diff
@@ -1,3 +1,26 @@
+{{- if .Values.extraCertificates }}
```

Comment:
`{{- if .Values.extraCertificates }}` is not needed here and can be removed.
If the list is empty - no op.